cli/cmd/clients.go

```diff
@@ -0,0 +1,27 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"github.com/cloudquery/plugin-sdk/clients"
+	"github.com/cloudquery/plugin-sdk/specs"
+	"github.com/rs/zerolog/log"
+)
+
+func initializeDestinationClients(ctx context.Context, sourceSpec specs.Source, destinationsSpecs []specs.Destination, cqDir string) ([]*clients.DestinationClient, error) {
+	var err error
```

Comment:
nit: let's also have the close logic here in case of an error. i.e this will return nil, err and close all clients in case one of them fails. + outside if it is not nil this will close all destinations - a bit of duplication but I think will make the code more streamlined/error-proof.

Reply:
Yeah, makes sense, I'll updated it 👍

Reply:
@yevgenypats I've refactored it a bit accordingly; rather than have the same code several times I felt it's a little nicer  to have a `destinationClients []*clients.DestinationClient` type with a `Close()` method.